SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -49,7 +57,7 @@ struct FConnectionConfig
 		{
```

Comment:
`void PreConnectInit(const bool bConnectAsClient)` Not the biggest fan of using bools that are not self evident (self evident: e.g. `void show(bool bShow)`) but that is really minor and maybe its preferable not to churn on the function signatures too much. So good with me.